wp-graphql.php

```diff
@@ -68,38 +100,27 @@ function graphql_can_load_plugin(): bool {
 		// IF WPGRAPHQL_AUTOLOAD is defined as false,
 		// but the WPGraphQL Class exists, we can assume the dependencies
 		// are loaded from the parent project.
-		if ( class_exists( '\WPGraphQL' ) ) {
-			return true;
-		}
+		return true;
 	}
 
-	// If the autoload file exists, load it
-	if ( file_exists( plugin_dir_path( __FILE__ ) . 'vendor/autoload.php' ) ) {
+	// @phpstan-ignore-next-line: this is ignored as the constant could be defined in wp-config, prior to being defined above
```

Comment:
Instead of ignoring the entire line, I think we can add `define( 'WPGRAPHQL_AUTOLOAD', true );` to `phpstan/constants.php`